### DISCUSSION TITLE
Finding Resources

### DISCUSSION THREAD

**Discussion by BobVillain** [community/community#21429] [Other Feature Feedback, Questions, & Ideas]
<p>Hi all, new to the community here. Nowhere near well-versed in writing code but I wanted to do something special for a friend.</p>
<p>Basically, I want to write something that I can input binary code into and then get written text as an output. So ideally, I can send the code to them, they run it, and then see a message.</p>
<p>Any pointers on where to find starting code?</p>
<p>Thanks!</p>

**Comment by Zerotask** [comment, marked as answer]
<p>Hey, this is a very basic translation and you can achieve this with almost every major programming language. Therefore this is a subjective recommendation: Use Go(lang) for it.<br>
This is a simple but yet powerful general purpose language by Google. It will produce a single executable which makes it very easy to send it to friends.</p>
<p>Here you can learn the basics of Go:<br>
<a href="https://play.golang.org" class="onebox" target="_blank" rel="noopener nofollow ugc">https://play.golang.org</a></p>
